project/Settings.scala

```diff
@@ -0,0 +1,83 @@
+import sbt._
+
+/**
+ * Application settings and dependencies
+ */
+object Settings {
+  object Definitions {
+    /** The name of the application */
+    val name = "ocs3"
+
+    /** Project version */
+    val version = "2016001.1.1"
```

Comment:
What's the benefit of `version.sbt` over putting it in the top level `build.sbt`?